src/server/command/worldedit/undo.zig

```diff
@@ -0,0 +1,23 @@
+const std = @import("std");
+
+const main = @import("root");
+const User = main.server.User;
+
+const Block = main.blocks.Block;
+const Blueprint = main.blueprint.Blueprint;
+
+pub const description = "Undo last change done to world with world editing commands.";
+pub const usage = "/undo";
+
+pub fn execute(args: []const u8, source: *User) void {
+	if(args.len != 0) {
+		source.sendMessage("#ff0000Too many arguments for command /undo. Expected no arguments.", .{});
+		return;
+	}
+	if(source.worldEditData.undoHistory.pop()) |action| {
+		action.blueprint.paste(action.position);
+		source.sendMessage("#00ff00Un-done last {s}.", .{action.message});
```

Comment:
Would it make sense to also print the position?

Reply:
Not really, it might be confusing, especially for brushes, cause brushes are applied from center while undo position for them will be min x min y min z. That reminds me that @min can be used on vectors.